ts/features/walletV3/payment/screens/WalletPaymentOutcomeScreen.tsx

```diff
@@ -1,28 +1,93 @@
-import { useNavigation } from "@react-navigation/native";
+import { Banner, VSpacer } from "@pagopa/io-app-design-system";
+import { openAuthenticationSession } from "@pagopa/io-react-native-login-utils";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { RouteProp, useNavigation, useRoute } from "@react-navigation/native";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
 import React from "react";
+import { View } from "react-native";
 import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import I18n from "../../../../i18n";
+import { mixpanelTrack } from "../../../../mixpanel";
 import {
   AppParamsList,
   IOStackNavigationProp
 } from "../../../../navigation/params/AppParamsList";
+import { useIOSelector } from "../../../../store/hooks";
+import { formatNumberCentsToAmount } from "../../../../utils/stringBuilder";
+import { WalletPaymentParamsList } from "../navigation/params";
+import { walletPaymentDetailsSelector } from "../store/selectors";
+import { WalletPaymentOutcome } from "../types/PaymentOutcomeEnum";
+
+type WalletPaymentOutcomeScreenNavigationParams = {
+  outcome: WalletPaymentOutcome;
+};
+
+type WalletPaymentOutcomeRouteProps = RouteProp<
+  WalletPaymentParamsList,
+  "WALLET_PAYMENT_OUTCOME"
+>;
 
 const WalletPaymentOutcomeScreen = () => {
+  const { params: _ } = useRoute<WalletPaymentOutcomeRouteProps>(); // TODO handle outcome
```

Comment:
Since in this PR, the failure/error outcome isn't managed yet, what do you think about adding also the ticket link to this TODO comment?

Reply:
Handled in https://github.com/pagopa/io-app/pull/5276/commits/c720daa3d933086195240a171ad9c7efd63e2811